docs/forms/FORM_ERRORS.md

```diff
@@ -74,15 +74,20 @@ export interface ErrorMessageObject {
   /**
    *  Translation key of the short error message (e.g. used for inline errors)
    *  @example
-   *  translationKey = 'travelerForm.firstName.required'; => corresponds to {'travelerForm.firstName.required': 'First name is required!'} in localization json;
+   *  ```typescript
```

Comment:
we are already in a `typescript` code block (line 71)

Reply:
Yes but this is a comment in the `typescript` block.
No problem on the interpretation and it is actually the what we want in this documentation.
This other proposal would be to remove this example but I am not sure to see a value in removing an example :S